### PR TITLE
browserslist-autofix: add new return with fixed for CHECKS

### PR DESCRIPTION
**Pull request by SergeyVinogorov**
Create new return with fixed key for lint function

### CODE REVIEW

**Comment by ai** [comment]
We need also change test to be sure that everything works fine

**Comment by ai** [comment]
Also, don’t forget to change docs

**Comment by SergeyVinogorov** [comment]
> 

about the documentation, did I understand correctly that need to update the changelog or did I miss something?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Change Log
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 0.2.2
```
Comment (ai) [inline CHANGELOG.md]: Please remove ChangeLog changes. It should `0.3`. And I add them in release commit.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "browserslist-lint",
-  "version": "0.2.1",
+  "version": "0.2.2",
```
Comment (ai) [inline package.json]: Revert it too

**Comment by ai** [comment]
> about the documentation, did I understand correctly that need to update the changelog or did I miss something?

You need to add API changes to `JS API` section of `README.md`

File: CHANGELOG.md (hunk 1 above)
Comment (SergeyVinogorov) [reply CHANGELOG.md]: Changed to 0.3

File: package.json (hunk 2 above)
Comment (SergeyVinogorov) [reply package.json]: Changed to 0.3.0 version, based on changelog.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
 # Change Log
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 0.3
```
Comment (ai) [inline CHANGELOG.md]: Revert these changes.

I change version only in release commit.

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "browserslist-lint",
-  "version": "0.2.1",
+  "version": "0.3.0",
```
Comment (ai) [inline package.json]: Revert it. I change version only in release commit.

**Comment by ai** [comment]
We also need to change docs:

https://github.com/browserslist/lint#js-api

File: CHANGELOG.md (hunk 3 above)
Comment (SergeyVinogorov) [reply CHANGELOG.md]: Ok, understood, pardon!
now i will check how works if set  zero queries i think there have bug with new return,
i'll fix it and make new commit again, thank you🤝

**Comment by SergeyVinogorov** [comment]
Reverted Changelog and version + make changes in docs in js-api

**Comment by ai** [comment]
Thanks. Released in 0.3.
